QStore4S/src/main/java/edu/asu/qstore4s/async/QueryInformation.java

```diff
@@ -0,0 +1,43 @@
+package edu.asu.qstore4s.async;
+
+public class QueryInformation {
+    private boolean status;
```

Comment:
I don't think that true/false is enough. What if there was an exception when the query was executed? You want to be able to tell the caller if the query is still running, if it was successfully completed, or if there were errors (and which ones).